src/cli.ts

```diff
@@ -245,21 +242,29 @@ program
     `List all locations to run the synthetics monitors. Pass optional '--url' and '--auth' to list private locations.`
   )
   .option('--url <url>', 'Kibana URL to fetch all public and private locations')
-  .option(
-    '--auth <auth>',
-    'API key used for Kibana authentication(https://www.elastic.co/guide/en/kibana/master/api-keys.html).'
-  )
+  .option('--auth <auth>', apiDocsLink)
   .action(async (cmdOpts: LocationCmdOptions) => {
+    const revert = installTransform();
+    let url = cmdOpts.url;
     try {
-      let locations = Object.keys(LocationsMap);
-      if (cmdOpts.auth && cmdOpts.url) {
-        const allLocations = await getLocations(cmdOpts);
-        locations = formatLocations(allLocations);
+      url = (await loadSettings())?.url;
```

Comment:
loadSettings will print error on stderr and cannot be caught, We need to fix this inorder to not print the message.

Reply:
not sure , what you mean? should i fix the function?

Reply:
if you run locations command now, you will see error printed on console saying no `synthetics.config.ts` file.